setup.py

```diff
@@ -57,7 +57,8 @@
     include_package_data=True,
     install_requires=[
         "numpy",
-        "ipywidgets>=7.6.0",
+        "ipywidgets>=7.6.0,<8.0.0",
+        "jupyterlab-widgets==1.1.1",
```

Comment:
Is it exactly that version or should it be `<=`?

Reply:
yup you're right, updated